v2/go.mod

```diff
@@ -1,9 +1,7 @@
 module github.com/storageos/go-api/v2
 
-go 1.15
```

Comment:
Do we want to remove this requirement? I am just not clear on what impact this has.

Reply:
Since the codegen is removing it and we don't have any specific golang version dependencies that I'm aware of, I'm going to leave it out.  Otherwise, I'll need to fix the codegen or remember to add it back in next time...